### PR TITLE
Add support for debugging flags in automated and manual tests

### PR DESCRIPTION
**Pull request by ma2ciek**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Feature: Added support for debugging flags in automated and manual tests available via the `--debug` (`-d`) flag. Closes ckeditor/ckeditor5#383.

---

### Additional information

Requires https://github.com/ckeditor/ckeditor5/pull/1912.

This PR adds support for the `--debug` (`-d`) flag in `yarn test` and `yarn manual` scripts. By default, it uncomments all lines starting with `// @if CK_DEBUG //`, but it also supports custom debugging flags. For example, the `yarn test --debug engine` will uncomment both `// @if CK_DEBUG //` and `// @if CK_DEBUG_ENGINE //`. The special `--debug false` turns off the default debugging and makes the `// @if CK_DEBUG //` statements uncommented.

### CODE REVIEW

File: packages/ckeditor5-dev-tests/lib/utils/ck-debug-loader.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+/**
+ * The loader matches sentences like: `// @if CK_DEBUG // someDebugCode();` and uncomment them.
+ * If also uncomments code after specific flags if they are provided to the webpack configuration.
```
Comment (Reinmar) [inline packages/ckeditor5-dev-tests/lib/utils/ck-debug-loader.js]:
```suggestion
 * It also uncomments code after specific flags if they are provided to the webpack configuration.
```

**Comment by Reinmar** [comment]
> Support for `--no-debug` should be nice. In such a case, `options.debug` is a boolean (`false`).

What would this option do?

**Comment by pomek** [comment]
Same what `"--debug false"` does. For blocking options via CLI, people are using `--no-*` syntax.

**Comment by pomek** [comment]
I'm also not sure about unlocking `--debug` options by default in automated tests. In the nearby future, we will have a solid mess in our build logs.

☝️ I'm assuming that we will have a lot of `console.log()` calls behind those debug flags.

**Comment by Reinmar** [comment]
I'm sorry, I don't know what this change does precisely. My guess is that we nee `--debug false` because `--debug` is on by default in our tools. Am I correct?

**Comment by pomek** [comment]
`--debug false` === `--no-debug`.

**Comment by Reinmar** [comment]
How does it answer my question? My question was about why do we need `--debug false` or `--no-debug`.

**Comment by ma2ciek** [comment]
@Reinmar, Yep, `debug` is turned on by default, like we've been discussing it in the https://github.com/ckeditor/ckeditor5/issues/383#issuecomment-505350323

@pomek I can add the support with:

**Comment by pomek** [comment]
Yes, it will.

**Pull request merged**